docs/sdk/py/client/index.mdx

```diff
@@ -0,0 +1,343 @@
+</head>
+
+# Client Module
+
+<a id="reduct.client"></a>
+
+# reduct.client
+
+Client module for ReductStore HTTP API
+
+<a id="reduct.client.Client"></a>
+
+## Client
+
+```python
+class Client()
+```
+
+HTTP Client for Reduct Storage HTTP API
+
+<a id="reduct.client.Client.__init__"></a>
+
+#### \_\_init\_\_
+
+```python
+def __init__(url: str,
+             api_token: Optional[str] = None,
+             timeout: Optional[float] = None,
+             extra_headers: Optional[Dict[str, str]] = None,
+             **kwargs)
+```
+
+Constructor
+
+**Arguments**:
+
+- `url` - URL to connect to the storage
+- `api_token` - API token if the storage uses it for authorization
+- `timeout` - total timeout for connection, request and response in seconds
+- `extra_headers` - extra headers to send with each request
+  Kwargs:
+- `session` - an external aiohttp session to use for requests
+- `verify_ssl` - verify SSL certificates
+
+**Example**:
+```python
+      client = Client("http://127.0.0.1:8383")
+      info = await client.info()
+   
+```
+
+<a id="reduct.client.Client.info"></a>
+
+#### info
+
+```python
+async def info() -> ServerInfo
+```
+
+Get high level server info
+
+**Returns**:
+
+  ServerInfo:
+  
+
+**Raises**:
+
+- `ReductError` - if there is an HTTP error
+
+<a id="reduct.client.Client.list"></a>
+
+#### list
+
+```python
+async def list() -> List[BucketInfo]
+```
+
+Return a list of all buckets on server
+
+**Returns**:
+
+  List[BucketInfo]
+
+**Raises**:
+
+- `ReductError` - if there is an HTTP error
+
+<a id="reduct.client.Client.get_bucket"></a>
+
+#### get\_bucket
+
+```python
+async def get_bucket(name: str) -> Bucket
+```
+
+Load a bucket to work with
+
+**Arguments**:
+
+- `name` - name of the bucket
+
+**Returns**:
+
+  Bucket
+
+**Raises**:
+
+- `ReductError` - if there is an HTTP error
+
+<a id="reduct.client.Client.create_bucket"></a>
+
+#### create\_bucket
+
+```python
+async def create_bucket(name: str,
+                        settings: Optional[BucketSettings] = None,
+                        exist_ok: bool = False) -> Bucket
+```
+
+Create a new bucket
+
+**Arguments**:
+
+- `name` - a name for the bucket
+- `settings` - settings for the bucket If None, the server
+  default settings is used.
+- `exist_ok` - the client raises no exception if the bucket
+  already exists and returns it
+
+**Returns**:
+
+- `Bucket` - created bucket
+
+**Raises**:
+
+- `ReductError` - if there is an HTTP error
+
+<a id="reduct.client.Client.get_token_list"></a>
+
+#### get\_token\_list
+
+```python
+async def get_token_list() -> List[Token]
+```
+
+Get a list of all tokens
+
+**Returns**:
+
+  List[Token]
+
+**Raises**:
+
+- `ReductError` - if there is an HTTP error
+
+<a id="reduct.client.Client.get_token"></a>
+
+#### get\_token
+
+```python
+async def get_token(name: str) -> FullTokenInfo
+```
+
+Get a token by name
+
+**Arguments**:
+
+- `name` - name of the token
+
+**Returns**:
+
+  Token
```

Comment:
The returns look different that argument and raises. Maybe we have something like:

- `Token`

Reply:
fixed in reduct-py